logging.rst

```diff
@@ -338,6 +338,16 @@ option of your handler to ``rotating_file``:
 Using a Logger inside a Service
 -------------------------------
 
+If your application uses :ref:`service autoconfiguration <services-autoconfigure>`,
+any service whose class implements :class:`Psr\Log\LoggerAwareInterface` will
```

Comment:
Double backslashes needed for the namespace inside :class:

Reply:
And this won't work, as it's not part of the `symfony/symfony` package